contract/contracts/services/EAS.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: CC-BY-NC-4.0
```

Comment:
Let's have this removed. The server and clients need typechain and abi to run so it's easier to deploy contract from there